src/components/Tokens/TokenDetails/TransactionsTable.tsx

```diff
@@ -0,0 +1,118 @@
+        disableSortBy: true,
+        id: ColumnHeader.Time,
+      },
+      {
+        Header: ColumnHeader.Type,
+        accessor: (swap) => swap,
+        Cell: ({ value }: { value: { input: SwapInOut } }) => {
+          const swapType = getSwapType(value.input.contractAddress, referenceTokenAddress)
+          return <TextCell color={getColor(swapType)}>{swapType}</TextCell>
+        },
+        disableSortBy: true,
+        id: ColumnHeader.Type,
+      },
+      {
+        Header: ColumnHeader.Amount,
+        accessor: (swap) => swap,
+        Cell: ({ value }: { value: { input: SwapInOut; output: SwapInOut } }) => {
+          const swapType = getSwapType(value.input.contractAddress, referenceTokenAddress)
+          const token = swapType === SwapAction.Buy ? value.output : value.input
+          return (
+            <TextCell color={getColor(swapType)}>{`${formatNumber({ input: token.amount, type: NumberType.TokenTx })} ${
+              token.symbol
+            }`}</TextCell>
+          )
+        },
+        disableSortBy: true,
+        id: ColumnHeader.Amount,
+      },
+      {
+        Header: ColumnHeader.For,
+        accessor: (swap) => swap,
+        Cell: ({ value }: { value: { input: SwapInOut; output: SwapInOut } }) => {
+          const swapType = getSwapType(value.input.contractAddress, referenceTokenAddress)
+          const token = swapType === SwapAction.Buy ? value.input : value.output
+          return (
+            <TextCell color={getColor(swapType)}>{`${formatNumber({ input: token.amount, type: NumberType.TokenTx })} ${
+              token.symbol
+            }`}</TextCell>
+          )
+        },
+        disableSortBy: true,
+        id: ColumnHeader.For,
+      },
+      {
+        Header: ColumnHeader.USD,
+        accessor: (swap) => swap,
+        Cell: ({ value }: { value: { input: SwapInOut; usdValue: number } }) => {
+          const swapType = getSwapType(value.input.contractAddress, referenceTokenAddress)
+          return (
+            <TextCell color={getColor(swapType)}>
+              {formatNumber({ input: value.usdValue, type: NumberType.PortfolioBalance })}
+            </TextCell>
+          )
+        },
+        disableSortBy: true,
+        id: ColumnHeader.USD,
+      },
+      {
+        Header: ColumnHeader.Maker,
+        accessor: 'maker',
+        Cell: ({ value }: { value: string }) => (
+          <LinkCell url={`https://etherscan.io/address/${value}`}>{shortenAddress(value)}</LinkCell>
+        ),
+        disableSortBy: true,
+        id: ColumnHeader.Maker,
+      },
+    ]
+  }, [theme.success, theme.critical, locale, referenceTokenAddress, formatNumber])
+  return (
+    <Table
+      columns={columns}
+      data={mockSwapData}
+      smallHiddenColumns={[ColumnHeader.For, ColumnHeader.USD, ColumnHeader.Maker]}
+      mediumHiddenColumns={[ColumnHeader.For, ColumnHeader.Maker]}
+      largeHiddenColumns={[ColumnHeader.Maker]}
+      extraLargeHiddenColumns={[]}
```

Comment:
`[]` not ideal here because it creates a new reference every render.. is there a `const STABLE_ARRAY = []` const somehwere?